pubspec.yaml

```
@@ -78,6 +79,9 @@ flutter:
         weight: 800
       - asset: assets/fonts/Pretendard-1.3.9/public/static/Pretendard-Black.otf
         weight: 900
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)